.github/workflows/nessie.yaml

```diff
@@ -445,47 +405,31 @@ jobs:
       TAG: ${{ needs.deploy-image.outputs.tag }}
       REPO: ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.us-east-1.amazonaws.com
     steps:
+      - name: Check-out code
```

Comment:
Why do we now need this step?  If it's for "Build lakectl for dbt container", why not put lakectl into the artifact?

Reply:
This was added in all steps because we need access to the new composite action